order-delivery/lib/models/Order.js

```diff
@@ -0,0 +1,53 @@
+const pool = require('../utils/pool');
+
+// static methods -> Order.insert(): Math.random(), Number.parseInt(), JSON.stringify()
+// instance methods -> arr.map(), params.get('code')
+module.exports = class Order {
+  id;
+  quantity;
+
+  constructor(row) {
+    this.id = row.id;
+    this.quantity = row.quantity;
+  }
+
+  static async insert(order) {
+    const {
+      rows,
+    } = await pool.query(
+      'INSERT INTO orders (quantity) VALUES ($1) RETURNING *',
+      [order.quantity]
+    );
+
+    return new Order(rows[0]);
+  }
+
+  static async produce() {
+    const { rows } = await pool.query('SELECT * from orders');
+
+    return rows.map((row) => new Order(row));
+  }
+
+  static async produceWithId(id) {
+    const { rows } = await pool.query('SELECT * from orders WHERE id=$1', [id]);
+
+    return rows.map((row) => new Order(row));
+  }
+
+  static async update(quantity, id) {
+    const {
+      rows,
+    } = await pool.query(
+      'UPDATE orders SET quantity=$1 WHERE id=$2 RETURNING *',
+      [quantity, id]
+    );
+    return new Order(rows[0]);
+  }
+
+  static async trash(id) {
+    console.log(id);
+    const { rows } = await pool.query('DELETE from orders WHERE id=$1', [id]);
```

Comment:
Here we want to add `RETURNING *` so we get back the item we just deleted. That should help with the test error. It looked like it was unable to find the id. :)